components/Pages/Trade/Swap/hooks/useSwap.ts

```diff
@@ -19,9 +20,9 @@ const useSwap = ({ reverse }) => {
   const { signingClient, cosmWasmClient } = useClients(walletChainName)
   const tokenA = useTokenInfo(swapTokenA?.tokenSymbol)
   const tokenB = useTokenInfo(swapTokenB?.tokenSymbol)
-  const slippage = useRecoilValue(slippageAtom)
+  let slippage = useRecoilValue(slippageAtom)
```

Comment:
possible to set default to1 1 or does it interfere with something else ?

Reply:
yes it changes the button from auto to custom with 1 as value